blocks/subscriptions-block/features/checkout/_children/BillingAddress/listOfStates.js

```diff
@@ -0,0 +1,212 @@
+const listOfStates = [
```

Comment:
@edwardcho1231 , do you mind check if we can use a library for this. Because here we are providing the states for USA, but we should support this for all countries.